QWeb/internal/text.py

```diff
@@ -104,13 +105,23 @@ def get_text_elements(text: str, **kwargs) -> Optional[list[WebElement]]:
             web_elements = _get_contains_text_element(text, **kwargs)
         except NoSuchFrameException:
             logger.debug("Got no such frame from contains text")
+    # find text directly from slots with supported parent elements
+    xpath_elements = None
+    slot_elements = get_slot_elements(text, partial)
```

Comment:
Define and initialize `slot_elements` at the top of the function for consistency

Reply:
handled with new commit